content/hardware/04.pro/boards/portenta-h7/tutorials/over-the-air-update/content.md

```diff
@@ -138,6 +138,52 @@ Convert your encoded file into `.ota` format
 
 You can use `OTA_Usage_Portenta.ino.PORTENTA_H7_M7` as a sketch name for facilitated identification of the file. After this, you will have the `.ota` file of the sketch that you will use with the OTA process. 
 
+### Installing Python 3 On Ubuntu 
```

Comment:
@Jorgetrujilloroman you are including a portenta H7 tutorial change that should not be on this PR.

Ask @marqdevx for help about how to solve this problem without removing this H7 tutorial from the server !

Reply:
Yup, we will split the commits on another branch :D